.github/workflows/ai-reviewer.yml

```diff
@@ -0,0 +1,29 @@
+name: Code Review
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    types: [opened, synchronize, review_requested]
```

Comment:
add a comment about what opened and synchronize are for? 
so they aren't removed later by accident